site/src/pages/tutorials/grpc/blog/implement-delete.mdx

```diff
@@ -0,0 +1,127 @@
+---
+menuTitle: "Implement DELETE"
+order: 6
+category: grpc
+tags:
+  - server
+level: basic
+type: step
+---
+
+# Implementing DELETE operation
+
+So far, we've created, read, and updated a blog post. Now, let's implement 
+and make a call to finally delete a blog post.
+
+<TutorialSteps current={6} />
+
+## What you need
+
+You need to have the files obtained from previous steps:
+
+- [Java code generated](/tutorials/grpc/blog/define-service#compile-proto-file)
+- Main.java
+- BlogService.java
+- BlogClient.java
+
+## 1. Implement service
+
+Let's implement a service method `deleteBlogPost()` in the BlogService.java we've been working on.
+
+1. Add a service method, `deleteBlogPost()`.
+```java filename=BlogService.java
+import example.armeria.server.blog.grpc.Blog.DeleteBlogPostRequest;
+
+public class BlogService extends BlogServiceGrpc.BlogServiceImplBase {
+  @Override
+  public void deleteBlogPost(DeleteBlogPostRequest request, StreamObserver<Empty> responseObserver) {
+    final BlogPost removed = blogPosts.remove(request.getId());
+    if (removed == null) {
+      responseObserver.onError(
+        throw new IllegalArgumentException("The post with ID": " + "does not exist");
+     } else {
+        responseObserver.onNext(Empty.getDefaultInstance());
+        responseObserver.onCompleted();
+     }
+ }
+}
+```
+2. Add Armeria's <type://GrpcServiceBuilder>'s blocking task executor to make all gRPC methods executed in the blocking task executor's thread pool.
```

Comment:
How about adding a brief explanation of why we need to set `useBlockingTaskExecutor()` to `true`?
For example:
```
By default, service methods are executed directly on the event loop for implementing fully asynchronous services. 
If your service uses blocking logic, you need to set `useBlockingTaskExecutor()` to `true`.  
```